vendor/vendor.json

```diff
@@ -99,6 +99,12 @@
 			"versionExact": "v10.2.1-beta"
 		},
 		{
+			"checksumSHA1": "/TBRYaTTjpsuKRCKH6nvQfSlLOw=",
+			"path": "github.com/Azure/azure-sdk-for-go/arm/operationalinsights",
+			"revision": "df4dd90d076ebbf6e87d08d3f00bfac8ff4bde1a",
+			"revisionTime": "2017-09-06T21:46:31Z"
```

Comment:
could we update this to pin to `v10.3.0-beta` of the Azure SDK for Go (to ensure it's consistent with the other vendored SDK's). You can do this by running:
```
govendor update github.com/Azure/azure-sdk-for-go/arm/operationalinsights@=v10.3.0-beta
```

Reply:
Done!